lms/validation/authentication/_bearer_token.py

```diff
@@ -111,10 +111,12 @@ def lti_user(self):
         :rtype: LTIUser
         """
         try:
-            return self.parse(locations=["headers", "querystring", "form"])
+            return self.parse(location=location)
         except ValidationError as error:
             try:
-                authorization_error_message = error.messages["authorization"][0]
+                authorization_error_message = error.messages[location]["authorization"][
+                    0
+                ]
```

Comment:
`error.messages["authorization"][0]` changed to `error.messages[location]["authorization"][0]`. Webargs has changed the format of its `error.messages` dicts, it used to be:

```python
{
  "authorization": ["error_message", ...]
}
```

It's now:

```python
{
  "headers": {
    "authorization": ["error_message", ...]
  }
}
```

where the `"headers"` key could be `"form"` or `"querystring"` or any other location, depending on the `location` arg that was used.